src/VaultSharp/VaultSharp.csproj

```diff
@@ -35,6 +35,12 @@
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Release\VaultSharp.XML</DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>VaultSharp.snk</AssemblyOriginatorKeyFile>
```

Comment:
... and just to be clear, the snk file I added was created using `sn -k VaultSharp.snk`.